UserSecrets.sln

```diff
@@ -1,23 +1,22 @@
-
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.22803.0
+VisualStudioVersion = 14.0.24709.0
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{01661652-A100-41E3-AE6E-E6151EB85666}"
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Extensions.Configuration.UserSecrets", "src\Microsoft.Extensions.Configuration.UserSecrets\Microsoft.Extensions.Configuration.UserSecrets.xproj", "{58B6443B-1278-4DF9-B7BB-DDF3BFFCF868}"
 EndProject
-Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "SecretManager", "src\Microsoft.Extensions.SecretManager\Microsoft.Extensions.SecretManager.xproj", "{8730E848-CA0F-4E0A-9A2F-BC22AD0B2C4E}"
-EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "test", "{9444A5DC-EE0B-4245-B81B-2CEF6F842F3D}"
 EndProject
-Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "SecretManager.Tests", "test\Microsoft.Extensions.SecretManager.Tests\Microsoft.Extensions.SecretManager.Tests.xproj", "{113EBBD2-E857-4CAF-9B53-7A8742CBCD4A}"
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Extensions.SecretManager.Tests", "test\Microsoft.Extensions.SecretManager.Tests\Microsoft.Extensions.SecretManager.Tests.xproj", "{113EBBD2-E857-4CAF-9B53-7A8742CBCD4A}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{CD87E04A-587D-44A6-BB15-A966DB3C18CF}"
 	ProjectSection(SolutionItems) = preProject
 		global.json = global.json
 	EndProjectSection
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "dotnet-user-secrets", "src\dotnet-user-secrets\dotnet-user-secrets.xproj", "{8730E848-CA0F-4E0A-9A2F-BC22AD0B2C4E}"
```

Comment:
Strange project name. Why make a change from `Microsoft.Extensions.SecretManage` as part of this PR?